OpenMined/syft
# feat(syftbox/client/plugins/apps.py): Replace default app run error m…

## Description
Replace default app run error message by one which actually provides the error message so the user can understand what was the problem

## Affected Dependencies
- CHANGED `syftbox/client/plugins/apps.py
`
## How has this been tested?
- Create an app that fails intentionally.
- Install it in syftbox
- Check if the error message is properly logged.
